mmv1/templates/terraform/examples/integrations_client_advance.tf.erb

```diff
@@ -0,0 +1,34 @@
+data "google_project" "test_project" {
+}
+
+resource "google_kms_key_ring" "keyring" {
+  name = "tf-test-keyring"
```

Comment:
let's try making the key ring and key names variables like here: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/data_fusion_instance_cmek.tf.erb#L14

this way we can avoid collision errors.

Reply:
Done. Added vars and referred that var here.